src/services/apes/apes.ts

```diff
@@ -83,7 +83,10 @@ export const apes = createApi({
       query: (endowId) => `${v(1)}/balances/${endowId}`,
     }),
     getStripePaymentStatus: builder.query<
-      Pick<PaymentIntent, "status"> & { guestDonor?: GuestDonor },
+      Pick<PaymentIntent, "status"> & {
+        guestDonor?: GuestDonor;
+        recipientName?: string;
```

Comment:
Note: won't work until https://github.com/AngelProtocolFinance/devops/pull/552 is merged and lambda code updated

Reply:
Merged, the Stripe flow is working now.